torch/csrc/jit/python/python_list.h

```diff
@@ -119,13 +119,13 @@ class ScriptList final {
   }
 
   // Get the size of the list.
-  diff_type len() const {
+  ssize_t len() const {
```

Comment:
Just double checked, it's size_t:
```
https://github.com/pybind/pybind11/blob/d72ffb448c58b4ffb08b5ad629bc788646e2d59e/include/pybind11/pytypes.h#L2065
```

Reply:
size_t is an unsigned type, and that causes some comparisons to fail because int64_t (signed 64 bit int) was used.